apisix/plugins/openid-connect.lua

```diff
@@ -89,6 +90,33 @@ local schema = {
             type = "string",
             default = "apisix",
         },
+        claim_validator = {
+            type = "object",
+            properties = {
+                audience = {
+                    type = "object",
+                    description = "audience claim value to validate",
+                    properties = {
+                        claim = {
+                            type = "string",
+                            description = "custom claim name",
+                            default = "aud",
+                        },
+                        required = {
+                            type = "boolean",
+                            description = "audience claim is required",
+                            default = false,
+                        },
+                        match_with_client_id = {
+                            type = "boolean",
+                            description = "audience must euqal to or includes client_id",
+                            default = false,
+                        }
+                    },
+                },
+            },
+            default = {},
```

Comment:
can we remove this default value?
empty table `{}`, it seems a little weird

Reply:
This can be remove, but `claim_validator` will not appear in configurations that have populated the defaults, nor will configuration items in `audience`, and the populating of the defaults is not recursive.
However, after checking, the code logic correctly handles the case where these config items don't exist.

The line will be removed.